detox/detox.d.ts

```diff
@@ -1287,18 +1287,23 @@ declare global {
         interface NativeElementActions {
             /**
              * Simulate tap on an element
-             * @param point relative coordinates to the matched element (the element size could changes on different devices or even when changing the device font size)
+             * @param point coordinates in the element's coordinate space. Optional (default is the center of the element).
              * @example await element(by.id('tappable')).tap();
              * @example await element(by.id('tappable')).tap({ x:5, y:10 });
              */
             tap(point?: Point2D): Promise<void>;
 
             /**
              * Simulate long press on an element
-             * @param duration (iOS only) custom press duration time, in milliseconds. Optional (default is 1000ms).
+             * @param duration custom press duration time, in milliseconds. Optional (defaults to the standard long-press duration for the platform).
+             * @param point coordinates in the element's coordinate space. Optional (default is the center of the element).
              * @example await element(by.id('tappable')).longPress();
+             * @example await element(by.id('tappable')).longPress(2000);
+             * @example await element(by.id('tappable')).longPress(1500, { x:5, y:10 });
+             * @example await element(by.id('tappable')).longPress({ x:5, y:10 });
              */
-            longPress(duration?: number): Promise<void>;
+            longPress(duration?: number, point?: Point2D): Promise<void>;
```

Comment:
reverse params order

Reply:
I'm torn between `point, duration` and by adding `longPress(params)`, where params is `{ x, y, duration }`. 😅

Reply:
`longPress(duration)` was already supported in iOS so we won't break this API, also I prefer to extend it in a flat way as we're doing with the rest of our Action APIs

Reply:
done - order replaced.

Reply:
now: `longPress(point, duration)`